firmware/octopi_firmware_v2/main_controller_teensy41/def_octopi_80120.h

```diff
@@ -17,30 +19,38 @@ static const bool flip_limit_switch_y = true;
 static const int FULLSTEPS_PER_REV_X = 200;
 static const int FULLSTEPS_PER_REV_Y = 200;
 static const int FULLSTEPS_PER_REV_Z = 200;
+static const int FULLSTEPS_PER_REV_W = 200;
 static const int FULLSTEPS_PER_REV_THETA = 200;
 
 float SCREW_PITCH_X_MM = 2.54;
 float SCREW_PITCH_Y_MM = 2.54;
 float SCREW_PITCH_Z_MM = 0.3;
+float SCREW_PITCH_W_MM = 1;
 
 int MICROSTEPPING_X = 256;
 int MICROSTEPPING_Y = 256;
 int MICROSTEPPING_Z = 256;
+int MICROSTEPPING_W = 64;
 
 static const float HOMING_VELOCITY_X = 0.8;
 static const float HOMING_VELOCITY_Y = 0.8;
 static const float HOMING_VELOCITY_Z = 0.5;
+static const float HOMING_VELOCITY_W = 0.15 * SCREW_PITCH_W_MM;
 
 long steps_per_mm_X = FULLSTEPS_PER_REV_X*MICROSTEPPING_X/SCREW_PITCH_X_MM;
 long steps_per_mm_Y = FULLSTEPS_PER_REV_Y*MICROSTEPPING_Y/SCREW_PITCH_Y_MM;
 long steps_per_mm_Z = FULLSTEPS_PER_REV_Z*MICROSTEPPING_Z/SCREW_PITCH_Z_MM;
+long steps_per_mm_W = FULLSTEPS_PER_REV_W*MICROSTEPPING_W/SCREW_PITCH_W_MM;
 
 float MAX_VELOCITY_X_mm = 50;
 float MAX_VELOCITY_Y_mm = 50;
 float MAX_VELOCITY_Z_mm = 2;
+float MAX_VELOCITY_W_mm = 3.19 * SCREW_PITCH_W_MM;
+
 float MAX_ACCELERATION_X_mm = 200;
 float MAX_ACCELERATION_Y_mm = 200;
 float MAX_ACCELERATION_Z_mm = 20;
+float MAX_ACCELERATION_W_mm = 300 * SCREW_PITCH_W_MM;
```

Comment:
@veerwang why are we multiplying the number with `SCREW_PITCH_W_MM`?

Reply:
I suppose the SCREW_PITCH_W_MM is 1. It means the squid filter spin a circle it move 1 unit. MAX_VELOCITY_W_mm multiply SCREW_PITCH_W_mm means the velocity spin 3.19 circles per-second. If some one change the SCREW_PITCH_W_mm value, it would change for MAX_VELOCITY_W_mm automatically.